### PR TITLE
Fix/Vue3 landmark WCAG fixes

### PR DESCRIPTION
**Pull request by bwennuh**
## What this does

Add some fixes for current WCAG issues on the Home (landing), Login, and Signup pages:

1. Update the html `lang` attribute to contain a value (`en`) in `index.html`
    - Axe Dev tools WCAG Issue Description: Ensures every HTML document has a lang attribute
 
2. Adds fixes for landmark tags in the Vue3 app HTML structure.
    - Axe Dev tools WCAG Issue Description 1: Ensures the document has a main landmark
    - Axe Dev tools WCAG Issue Description 2: Ensures all page content is contained by landmarks

## Checklist
- [x] Update the `<html>` tag for the Vue3 `index.html` file to be `<html lang="en">`
- [x] Add a `<main>` tag for the main content to be display on the page
- [x] Add a `<nav>` tag for the navigation content + links to be displayed at the top of the screen

## How to test

If you have the axe dev tools extension in Chrome, you can use them to scan the page and verify that Home, Login, and Sign Up pages do not contain issues related to WCAG guidelines.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/clients/vue3/public/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="en">
```
Comment (bwennuh) [inline {{cookiecutter.project_slug}}/clients/vue3/public/index.html:2]: **Info:** Needed a language defined here for the `lang` attribute for WCAG compliance.

Screen shot of issue from Axe Dev tools in Chrome attached.
Screen Shot 2023-01-04 at 5 30 42 PM

File: {{cookiecutter.project_slug}}/clients/vue3/src/App.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="font-avenir antialiased text-center">
-    <div class="flex justify-center p-7">
+    <nav class="flex justify-center p-7">
```
Comment (bwennuh) [inline {{cookiecutter.project_slug}}/clients/vue3/src/App.vue:3]: **Info:** Changed the `<div>` tag to a `<nav>` tag to comply with the landmark requirements as defined by WCAG for compliance.

Screen shot of issue from Axe Dev tools in Chrome attached below:

Screen Shot 2023-01-04 at 5 31 26 PM

File: {{cookiecutter.project_slug}}/clients/vue3/src/App.vue [hunk 3]
```diff
@@ -14,8 +14,10 @@
         |
         <router-link to="/signup" class="router">Signup</router-link>
       </template>
-    </div>
-    <router-view />
+    </nav>
+    <main>
+      <router-view />
+    </main>
```
Comment (bwennuh) [inline {{cookiecutter.project_slug}}/clients/vue3/src/App.vue:20]: **Info:** Added a `<main>` tag wrapper for the router-view in order to comply with the main body content landmark requirements as defined by WCAG.

Screen shot of issue from Axe Dev tools in Chrome attached below:

File: {{cookiecutter.project_slug}}/clients/vue3/src/views/Dashboard.vue [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <template>
-  <div class="dashboard">
-    <h2>This is your dashboard</h2>
+  <div>
+    <h1 class="heading">This is your dashboard</h1>
```
Comment (bwennuh) [inline {{cookiecutter.project_slug}}/clients/vue3/src/views/Dashboard.vue:3]: **Info:** Heading tags should start in hierarchy order, starting with `<h1>`. This helps users with screen readers and other assistive technology to appropriately find information on the page.

If specific is styling is needed for a heading tag, the CSS should be updated (vs using a different heading # tag).

**Comment by bwennuh** [comment]
For reference, the above fixes will now structure the DOM for the Vue3 side to look as follows:
(Note: the green underlines are to help indicate the main changes with this PR)

Screen Shot 2023-01-04 at 5 28 19 PM

**Review by oudeismetis — APPROVED** [review]

**Pull request merged**